meudesafio/page.html

```diff
@@ -7,19 +7,19 @@
 </head>
 
 <body>
```

Comment:
@marianaruddy a tag **<body>** não está sendo fechada lá embaixo... dê uma olhada depois

```suggestion
<body>

(conteúdo das demais sections)

</body>
```